### ISSUE TITLE
`vim.treesitter.get_query() is deprecated` warning when fuzzy searching

### ISSUE THREAD

**Issue by krlenell** [nvim-lua/kickstart.nvim#277]
## Describe the bug

When attempting to fuzzy search by typing the spacebar followed by '/' this warning appears. This happened after running `brew update` and `brew upgrade` earlier. Fuzzy searching can still be done, but the alert appears anyways.

Here is the full warning:

```
vim.treesitter.get_query() is deprecated, use vim.treesitter.query.g
et_query() instead. :help deprecated
stack traceback:
        ...r/neovim/0.9.0/share/nvim/runtime/lua/vim/treesitter.lua:
28: in function '__index'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:
435: in function 'v'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:
570: in function 'v'
        .../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:
511: in function 'current_buffer_fuzzy_find'
        /Users/name/.config/nvim/init.lua:279: in function </Users/be
vim.treesitter.query.get_query() is deprecated, use vim.treesitter.q
uery.get() instead. :help deprecated
stack traceback:
        ...im/0.9.0/share/nvim/runtime/lua/vim/treesitter/query.lua:
200: in function <...im/0.9.0/share/nvim/runtime/lua/vim/treesitter/
query.lua:199>
        [C]: in function 'pcall'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:
435: in function 'v'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:
570: in function 'v'
        .../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:
511: in function 'current_buffer_fuzzy_find'
        /Users/name/.config/nvim/init.lua:279: in function </Users/be
p/.config/nvim/init.lua:277>
Press ENTER or type command to continue
```

Specified lines in init.lua:
```
vim.keymap.set('n', '<leader>/', function()
  -- You can pass additional configuration to telescope to change theme, layout, etc.
  require('telescope.builtin').current_buffer_fuzzy_find(require('telescope.themes').get_dropdown {
    winblend = 10,
    previewer = false,
  })
end, { desc = '[/] Fuzzily search in current buffer' })
```

## To Reproduce

1. Likely need to update treesitter and nvim
2. Press space
3. press `/`

## Desktop

- OS: 13.2.1 (22D68)
- Terminal: iTerm or Terminal

## Neovim Version

```
NVIM v0.9.0
Build type: Release
LuaJIT 2.1.0-beta3
   system vimrc file: "$VIM/sysinit.vim"
   fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.9.0/share/nvim
```

**Comment by Drakx** [comment]
Try updating Telescope, https://github.com/nvim-telescope/telescope.nvim/pull/2451 looks as though this has been fixed?

**Comment by feoh** [comment]
Yup. Please try what @Drakx suggested and feel free to re-open/file a new issue if that doesn't work.

**Comment by krlenell** [comment]
I updated treesitter. It does not seem to work. 
<img width="871" alt="image" src="https://user-images.githubusercontent.com/61432718/234758812-2987db06-b940-4b42-9f91-ac6f0086b470.png">

**Comment by krlenell** [comment]
telescope.nvim seems to be up to date and I still have the problem
<img width="519" alt="image" src="https://user-images.githubusercontent.com/61432718/234759447-e61d6a53-de0c-4cde-a03f-091b5159a566.png">

**Comment by 0xDmtri** [comment]
> telescope.nvim seems to be up to date and I still have the problem <img alt="image" width="519" src="https://user-images.githubusercontent.com/61432718/234759447-e61d6a53-de0c-4cde-a03f-091b5159a566.png">

Yes, same here as well

**Comment by Drakx** [comment]
Strange, try backing up your current config and reclone the repo. I think it could be a caching issue, though I don't think it'll make much difference.

**Comment by feoh** [comment]
Ooh! Good point.

You might try:

`brew uninstall neovim`

rm -rf ~/.local/share/nvim

That'll nuke your cache. Then reinstall and re-run.

**Comment by 0xDmtri** [comment]
> Ooh! Good point.
> 
> You might try:
> 
> `brew uninstall neovim`
> 
> rm -rf ~/.local/share/nvim
> 
> That'll nuke your cache. Then reinstall and re-run.

didn't work ((

**Comment by Drakx** [comment]
Try `rm -rf ~/.local/share/nvim ~/.local/state/nvim ~/.cache/nvim` this will basically rest neovim to "out the box" experience
Make sure you have a backup of any changes you've made before you do this command

**Comment by 0xDmtri** [comment]
> Try `rm -rf ~/.local/share/nvim ~/.local/state/nvim ~/.cache/nvim` this will basically rest neovim to "out the box" experience Make sure you have a backup of any changes you've made before you do this command

same thing unfortunately, tested on both macs, full delete of aforementioned dirs + new git clone.

**Comment by krlenell** [comment]
I tried everything suggested here and updated plugins in Lazy after reinstalling and was unable to fix the bug and succeeded in getting what seems to be a larger error message. 

```
vim.treesitter.get_query() is deprecated, use vim.treesitter.query.get_query() instead. :help deprecated
stack traceback:
        ...r/neovim/0.9.0/share/nvim/runtime/lua/vim/treesitter.lua:28: in function '__index'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:435: in function 'v'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:570: in function 'v'
        .../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:511: in function 'current_buffer_fuzzy_find'
vim.treesitter.query.get_query() is deprecated, use vim.treesitter.query.get() instead. :help deprecated
stack traceback:
        ...im/0.9.0/share/nvim/runtime/lua/vim/treesitter/query.lua:200: in function <...im/0.9.0/share/nvim/runtime/lua/vim/treesitter/query.lua:199>
        [C]: in function 'pcall'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:435: in function 'v'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:570: in function 'v'
        .../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:511: in function 'current_buffer_fuzzy_find'
        /Users/user/.config/nvim/init.lua:277: in function </Users/user/.config/nvim/init.lua:275>
 ```

**Comment by 0xDmtri** [comment]
> 

yeah i get the same one...

**Comment by declan-whiting** [comment]
I've just opened a PR which worked for me. 

When I jumped in to ~/.local/share/nvim/lazy/telescope.nvim and ran a git log, I could see that the latest commit was from January and didn't include the fix for the deprecated functions.  **(Consistent with the latest release tag)**

Initially I just checked out master as a smoke test and fixed the issue. But that is not recommend.  So #303  as per the guidance uses the **latest release branch**
